blocks/post-sidebar/post-sidebar.css

```diff
@@ -0,0 +1,153 @@
+/* SIDE BAR */
+
+.post .post-sidebar-wrapper {
```

Comment:
i'd recommend moving this, as well as the two below that use .post-sidebar-container into post.css. Reasoning is that those are controlling the post grid, and not the sidebar itself, so just a cleaner separation

Reply:
changed and re-commited